tests/deprecations_ignore.yml

```diff
@@ -395,6 +395,9 @@
 - tests/providers/google/cloud/hooks/test_bigquery.py::TestTimePartitioningInRunJob::test_run_query_with_arg
 - tests/providers/google/cloud/hooks/test_bigquery.py::TestTimePartitioningInRunJob::test_run_with_auto_detect
 - tests/providers/google/cloud/hooks/test_cloud_sql.py::TestCloudSqlDatabaseHook::test_cloudsql_database_hook_get_database_hook
+- tests/providers/google/cloud/hooks/test_datapipeline.py::TestDataPipelineHook::test_get_conn
+- tests/providers/google/cloud/hooks/test_datapipeline.py::TestDataPipelineHook::test_create_data_pipeline
+- tests/providers/google/cloud/hooks/test_datapipeline.py::TestDataPipelineHook::test_run_data_pipeline
```

Comment:
I think the aim of this list is to be shorter over time and not bigger.
Why do we need to ignore the deprecations? it's best to just handle them

Reply:
They were added to this list to not raise exceptions in the tests, but just warnings instead

Reply:
@Taragolis hi! What do you think about this? Is it relevant change?

Reply:
thank you for suggestion, I have modified the code :)